src/components/pickers/SuccessPickerDrawer.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Drawer, Button } from "@mui/material";
+import IconPicker from '../../../assets/icons/picker.svg';
+
+
+interface SuccessDrawerProps {
+  open: boolean;
+  dismiss: () => void;
+}
+
+const SuccessDrawer: React.FC<SuccessDrawerProps> = ({ open, dismiss }) => {
+  return (
+    <Drawer anchor="right" open={open} onClose={dismiss}>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          margin: "auto",
+        }}
+      >
+        <img
+          src={IconPicker}
+          alt="Icon Picker"
+          style={{
+            width: "221px",
+            height: "229px",
+            marginTop: "16px",
+          }}
+        />
+        <div style={{ width: 600, padding: 20 }}>
+          <p
```

Comment:
Let's see what Typography component matches this config and use it. They could be Display, BodyText and Labl